state/kafka-receiver/src/test/java/com/expediagroup/streamplatform/streamregistry/state/kafka/KafkaEventReceiverTest.java

```diff
@@ -141,6 +159,62 @@ public void listenerThrowsException() throws Exception {
     inOrder.verify(correlator).received("foo");
   }
 
+  @Test
+  public void errorWhenWrongPartitionCount() {
+    when(config.getTopic()).thenReturn(topic);
+    val topicPartitionsList = new ArrayList<PartitionInfo>();
+    topicPartitionsList.add(partitionInfo);
+    topicPartitionsList.add(partitionInfo);
```

Comment:
Is this supposed to be here twice?

Reply:
Yes. `KafkaEventReceiver` only supports single partitioned topics, so this is the only way I could mock it as a multi partitioned topic.

But I think the fact you asked the question means I need to make this a bit more obvious 😄

Reply:
Hopefully a bit clearer now 😄